apps/studio/src/pages/sites/[siteId]/pages/[pageId]/index.tsx

```diff
@@ -51,15 +51,17 @@ function EditPage(): JSX.Element {
       </GridItem>
       {/* TODO: Implement preview */}
       <GridItem colSpan={2} overflow="scroll">
-        {/* TODO: the version here should be obtained from the schema  */}
-        {/* and not from the page */}
-        <Preview
-          siteId={siteId}
-          {...page}
-          permalink={permalink}
-          version="0.1.0"
-          content={previewPageState}
-        />
+        <Box p="2rem" bg="gray.100">
+          <Box borderRadius="8px" bg="white" shadow="md" overflow="hidden">
```

Comment:
will the hidden overflow cause the preview to not be able to be scrolled?
should it be auto instead?

Reply:
lemme try this - i put `overflow="hidden"` because otherwise, the overflow would cause the `borderRadius` to not work properly (the masthead wouldn't have border :sadge:)

Reply:
tested against local - `auto` works, i think the reason is just that the default is `visible` and we just need to constrain the masthead into the `Box`. 

thanks for catching!